plugins/database/plugin.go

```diff
@@ -77,6 +78,17 @@ func configure(_ *node.Plugin) {
 		log.Fatalf("Failed to check database version: %s", err)
 	}
 
+	if str := config.Node().GetString(CfgDatabaseDirty); str != "" {
+		val ,err := strconv.ParseBool(str)
```

Comment:
The dog is right. It's not `gofmt`-ed. Best set up `goimports` as a file watcher in your IDE and this shouldn't happen at all.